app/jobs/send_request_events_to_bigquery.rb

```diff
@@ -0,0 +1,10 @@
+class SendRequestEventsToBigquery < ApplicationJob
+  def perform(request_event_json)
+    return unless FeatureService.enabled?(:send_request_data_to_bigquery)
```

Comment:
Just thinking about the alternative branch here, where the feature isn't enabled. I'm assuming that the SideKiq job will be dropped at this point?